frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx

```diff
@@ -10,12 +10,59 @@ import { InfoCircleOutlined } from '@ant-design/icons'
 import { trendsLogic } from '../../trendsLogic'
 import { ViewType } from '../../insightLogic'
 import { LIFECYCLE } from 'lib/constants'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 
-export function TrendTab(): JSX.Element {
-    const { filters } = useValues(trendsLogic({ dashboardItemId: null, view: ViewType.TRENDS }))
-    const { setFilters } = useActions(trendsLogic({ dashboardItemId: null, view: ViewType.TRENDS }))
+interface TrendTabProps {
+    view: string
+}
+
+export function TrendTab({ view }: TrendTabProps): JSX.Element {
+    const { filters } = useValues(trendsLogic({ dashboardItemId: null, view }))
+    const { setFilters } = useActions(trendsLogic({ dashboardItemId: null, view }))
+    const { featureFlags } = useValues(featureFlagLogic)
 
-    return (
+    return featureFlags['remove-shownas'] ? (
+        <>
+            <h4 className="secondary">{'Actions & Events'}</h4>
+            <ActionFilter
+                filters={filters}
+                setFilters={(payload): void => setFilters(payload)}
+                typeKey="trends"
+                hideMathSelector={filters.shown_as === LIFECYCLE}
+                copy="Add graph series"
+                disabled={filters.shown_as === LIFECYCLE && (filters.events?.length || filters.actions?.length)}
+            />
+            <hr />
+            <h4 className="secondary">Filters</h4>
+            <PropertyFilters pageKey="trends-filters" />
+            {filters.insight === ViewType.TRENDS && <hr />}
+            {filters.insight === ViewType.TRENDS && (
+                <h4 className="secondary">
+                    Break down by
+                    <Tooltip
+                        placement="right"
+                        title="Use breakdown to see the volume of events for each variation of that property. For example, breaking down by $current_url will give you the event volume for each url your users have visited."
```

Comment:
Not sure if this is this is within scope of this PR, but I find this a tad confusing because a) we're referencing the "volume events" when I could be doing active users, mean, P90, ...; and b) we say `$current_url` when these properties are shown in a more human-friendly way in the list (i.e. `$current_url` is actually shown as "Current URL")

Reply:
tweaked the copy a bit. still seems confusing lmk if you have a better fix